cmd/minikube/cmd/delete.go

```diff
@@ -42,17 +43,14 @@ associated files.`,
 			os.Exit(1)
 		}
 
-		scanner := bufio.NewScanner(os.Stdin)
-		var text string
-		for !(text == "Y" || text == "y") {
-			fmt.Print(`Are you sure you want to delete Minikube? It will delete the whole
-cluster, all data will be lost. [y/N] `)
-			scanner.Scan()
-			text = scanner.Text()
-			if text == "N" || text == "n" || text == "" {
-				fmt.Println("Minikube deletion is aborted.")
-				os.Exit(1)
-			}
+		confirmPrompt := "Are you sure you want to delete Minikube? It will delete the whole cluster, all data will be lost."
+		posResponses := []string{"y", "yes", "Y", "Yes"}
```

Comment:
IIRC there is no need for the capitalized version; `AskForYesNoConfirmation` internally does a `ToLower` AFAIR.

Reply:
what if I typed `YES` or `NO`. I believe the function already handles this.

Reply:
Updated, thanks